colcon_test_result/verb/test_result.py

```diff
@@ -56,23 +56,22 @@ def main(self, *, context):  # noqa: D102
                     print(result.path)
                 else:
                     print(result)
+                    if context.args.verbose:
+                        for detail in result.details:
+                            for i, line in enumerate(detail.splitlines()):
+                                print('-' if i == 0 else ' ', line)
 
-                if context.args.verbose:
-                    for detail in result.details:
-                        for i, line in enumerate(detail.splitlines()):
-                            print('-' if i == 0 else ' ', line)
-
-        if (
-            any(r.error_count or r.failure_count for r in results) or
-            (context.args.all and results)
-        ):
-            print()
 
         summary = Result('Summary')
         for result in results:
             summary.add_result(result)
 
         if not context.args.result_files_only:
+            if (
+                    any(r.error_count or r.failure_count for r in results) or
+                    (context.args.all and results)
```

Comment:
Nitpick: these two lines should be indented one level (4 spaces from the `if`).

Reply:
Weird. My IDE (PyCharm) did not like that indentation. I put it back the way you like it.

Reply:
It is less my "whim" but more like what PEP 8 defines...